core/runtime/src/main/java/io/quarkus/runtime/Timing.java

```diff
@@ -79,6 +81,10 @@ public static void printStopTime(String name) {
                 (UNSET_VALUE.equals(name) || name == null || name.trim().isEmpty()) ? "Quarkus" : name,
                 secondsRepresentation);
         bootStopTime = -1;
+        /**
+         * We can safely close log handlers after stop time has been printed.
+         */
+        InitialConfigurator.DELAYED_HANDLER.close();
```

Comment:
I would instead do something like:

```java
for (Handler h : InitialConfigurator.DELAYED_HANDLER.clearHandlers()) {
    try {
        h.close();
    } catch (Throwable ignored) {
    }
}
```

This way a subsequent dev mode iteration is more likely to work correctly.

Reply:
Okay, thanks for the feedback David. Let me do the change.